<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>login_window</class>
    <widget class="QWidget" name="login_window">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>804</width>
                <height>666</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Login</string>
        </property>
        <property name="styleSheet">
            <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0" columnstretch="0,0,0,0">
            <property name="leftMargin">
                <number>100</number>
            </property>
            <property name="topMargin">
                <number>100</number>
            </property>
            <property name="rightMargin">
                <number>100</number>
            </property>
            <property name="bottomMargin">
                <number>100</number>
            </property>
            <property name="spacing">
                <number>20</number>
            </property>
            <item row="4" column="3">
                <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>40</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="6" column="2">
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="1" column="2">
                <widget class="QLabel" name="login_label">
                    <property name="maximumSize">
                        <size>
                            <width>1000</width>
                            <height>300</height>
                        </size>
                    </property>
                    <property name="font">
                        <font>
                            <weight>50</weight>
                            <bold>false</bold>
                        </font>
                    </property>
                    <property name="styleSheet">
                        <string notr="true"/>
                    </property>
                    <property name="text">
                        <string>Login</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item row="5" column="2">
                <widget class="QPushButton" name="login_button">
                    <property name="font">
                        <font>
                            <weight>50</weight>
                            <bold>false</bold>
                        </font>
                    </property>
                    <property name="styleSheet">
                        <string notr="true"/>
                    </property>
                    <property name="text">
                        <string>Login</string>
                    </property>
                </widget>
            </item>
            <item row="4" column="2">
                <widget class="QLineEdit" name="login_input">
                    <property name="maximumSize">
                        <size>
                            <width>250</width>
                            <height>16777215</height>
                        </size>
                    </property>
                    <property name="font">
                        <font>
                            <weight>50</weight>
                            <bold>false</bold>
                        </font>
                    </property>
                    <property name="layoutDirection">
                        <enum>Qt::LeftToRight</enum>
                    </property>
                    <property name="styleSheet">
                        <string notr="true"/>
                    </property>
                    <property name="placeholderText">
                        <string>Password</string>
                    </property>
                </widget>
            </item>
            <item row="4" column="0">
                <spacer name="horizontalSpacer">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>40</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item row="0" column="2">
                <spacer name="verticalSpacer_2">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
